src/modules/objects/objectStore.js

```diff
@@ -0,0 +1,41 @@
+import { createStore, createAction } from 'react-rxjs/dist/RxStore';
+import { Observable } from 'rxjs';
+import MusitObject from '../../models/object';
+import Sample from '../../models/sample';
+import Event from '../../models/event';
+export const initialState = { objectData: {}, events: [], samples: [] };
+
+const loadObjectData = () =>
+  val => {
+    return MusitObject.getObjectDetails()(val)
```

Comment:
I find it easier to read when we drop the return and change {} to (). `val => ( MusitObject.getObjectDetails()(val) ... )`